x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -108,6 +108,34 @@ describe('when on the list page', () => {
     });
   });
 
+  describe('when loading data with the query_strategy_version is `v1`', () => {
+    beforeEach(() => {
+      reactTestingLibrary.act(() => {
+        const mockedEndpointListData = mockEndpointResultList({
+          total: 4,
+          query_strategy_version: MetadataQueryStrategyVersions.VERSION_1,
+        });
+        setEndpointListApiMockImplementation(coreStart.http, {
+          endpointsResults: mockedEndpointListData.hosts,
+          queryStrategyVersion: mockedEndpointListData.query_strategy_version,
+        });
+      });
+    });
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+    it('should not display the KQL bar', async () => {
+      const renderResult = render();
+      await reactTestingLibrary.act(async () => {
+        await middlewareSpy.waitForAction('serverReturnedEndpointList');
+      });
+      // const adminSearchBar = await renderResult.findByTestId('adminSearchBar');
+      return renderResult.findByTestId('adminSearchBar').catch((e) => {
```

Comment:
Alternative would be to use the `queryByTestId()` which will not throw, but will instead return `null` if the element is not found:

```suggestion
      expect(renderResult.queryByTestId('adminSearchBar')).toBeNull();
```